pkgs/games/minetest/default.nix

```diff
@@ -35,7 +38,7 @@ let
       "-DENABLE_FREETYPE=1"
       "-DENABLE_GETTEXT=1"
       "-DENABLE_SYSTEM_JSONCPP=1"
-      "-DIRRLICHT_INCLUDE_DIR=${irrlicht}/include/irrlicht"
+      "-DIRRLICHT_INCLUDE_DIR=${irrlichtmt}/include/irrlichtmt"
```

Comment:
Since Minetest 5.5.0 can be compiled with touch support, can we add an option for that? `ENABLE_TOUCH` is the flag according to the README at https://github.com/minetest/minetest.

Reply:
I added `withTouchSupport`, which also requires rebuilding IrrlichtMT (upstream is not set up to build with touchscreen support by default), but I did test it on my Thinkpad X200 Tablet (which is the one x86 touchscreen device running NixOS I have) and it works with multitouch support and everything.